src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbstractClassNameCheck.java

```diff
@@ -64,8 +79,26 @@
  * <p>Example:</p>
  * <pre>
  * abstract class AbstractFirstClass {} // OK
- * abstract class SecondClass {} // violation, it should match the pattern "^Abstract.+$"
+ * abstract class SecondClass {} // violation, it should match pattern "^Abstract.+$"
  * class AbstractThirdClass {} // OK, no "abstract" modifier
+ * class FourthClass {} // OK
+ * </pre>
+ * <p>
+ * The following example shows how to configure the {@code AbstractClassName} to
+ * checks names, but ignore {@code Name} modifiers:
```

Comment:
please add one more example config and codeblock for `format` property.
We need example for all properties.

Reply:
sure

Reply:
can you please help me that how can I do it. Do I need make changes here only or make changes again locally and then push it.

Reply:
yes, please push changes, I need them to see in this PR.
So it will be then be merged and availbe on web side after merge

Reply:
@romani I pushed changes to my repository https://github.com/pkpale/checkstyle.git
but I don't know how to get them into this PR. Can you please help me with it.